test/persistence-hooks.test.js

```diff
@@ -4,9 +4,11 @@
 // License text available at https://opensource.org/licenses/MIT
 
 'use strict';
-var should = require('./init');
-var suite = require('loopback-datasource-juggler/test/persistence-hooks.suite.js');
+describe('persistence-hooks suite test', function() {
+  before(function() {
+    require('./init.js');
+  });
 
-suite(global.getDataSource(), should, {
-  replaceOrCreateReportsNewInstance: true,
+  require('loopback-datasource-juggler/test/persistence-hooks.suite.js');
```

Comment:
The file 'loopback-datasource-juggler/test/persistence-hooks.suite.js' is required but not executed here.
With `describe()` it should be something like:
```js
'use strict';
describe('persistence-hooks suite test', function() {
  var should = require('./init.js');
  var suite = require('loopback-datasource-juggler/test/persistence-hooks.suite.js');
  
  suite(global.getDataSource(), should, {replaceOrCreateReportsNewInstance: true});
});
```